content/2018/01/maxmind-and-shopify-to-speak-about-account-takeover-at-mrc-vegas.md

```diff
@@ -41,7 +41,7 @@ possible._
 
 * * *
 
-## **Important MRC Vegas Update:**
+## **Important MRC Vegas Update**
```

Comment:
We seem to bold headings quite a bit. Is there a reason for this? Aren't they already bold by virtue of being a heading? I wonder if this should be addressed in the styling if there is a reason that we do this.

Reply:
I don't think we ever really linted or did a deep dive of updating the content when we switched from WP to Gatsby so there is some odd styling especially for the older posts. I made some judgment calls for most changes, but I think for this branch I might just keep the existing content changes and we can defer to marketing on updating the rest of the content according to your comments. Does that work?

Reply:
That is fine. It'd probably be easy to update all of these automatically though with `sed` or a Perl one liner. If there isn't a styling reason for doing this, I could push the changes to your branch.

Reply:
Sure, pushing that change makes sense.